File: exercise/C#/day23/ControlSystem.Core/AmplifierType.cs
namespace ControlSystem.Core
{
    public enum AmplifierType
    {
        Basic = 1,
        Blessed = 2,
        Divine = 3
    }

    public static class AmplifierTypeExtensions
    {
        public static int GetMultiplier(this AmplifierType amplifierType) => (int) amplifierType;
    }
}

File: solution/day23/src/main/java/core/control/generator/AmplifierType.java
package core.control.generator;

public enum AmplifierType {
    BASIC(1),
    BLESSED(2),
    DIVINE(3);

    private final int multiplier;

    AmplifierType(int multiplier) {
        this.multiplier = multiplier;
    }

    public int getMultiplier() {
        return multiplier;
    }
}
